source/chapters/controlling_mixxx.rst

```diff
@@ -166,7 +166,7 @@ folder in your :term:`OS <operating system>` file browser:
 
   * GNU/Linux: :file:`/home/<username>/.mixxx/controllers`
   * macOS: :file:`/Users/<username>/Library/Application Support/Mixxx/controllers`
-  * Windows Vista and newer: :file:`C:\\Users\\<username>\\AppData\\Mixxx\\controllers`
+  * Windows Vista and newer: :file:`C:\\Users\\<username>\\AppData\\Local\\Mixxx\\controllers`
```

Comment:
while this may work better for most users (who notice to replace `username` ;) we could also use the paths from the wiki
https://github.com/mixxxdj/mixxx/wiki/Testing#backup-settings-and-database